### PR TITLE
[16.0] [MIG] company_today

### PR DESCRIPTION
**Pull request by victor-champonnois**
https://gestion.coopiteasy.be/web#id=10959&action=475&active_id=492&model=project.task&view_type=form&menu_id=

### CODE REVIEW

File: company_today/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "application": False,
+    "depends": [
+        "base",
+    ],
+    "excludes": [],
+    "data": [
+        "data/ir_cron.xml",
+    ],
+    "demo": [],
+    "qweb": [],
```
Comment (huguesdk) [inline company_today/__manifest__.py]:
```suggestion
    "depends": [
        "base",
    ],
    "data": [
        "data/ir_cron.xml",
    ],
```

default and empty values should ideally be removed.

**Review by huguesdk — APPROVED** [review]
lgtm! just a small suggestion (not related to the porting).

**Review by carmenbianca — APPROVED** [review]
same comment as hugues

**Pull request merged**